src/test/scala/nl.knaw.dans.easy.solr4files/SolrErrorHandlingSpec.scala

```diff
@@ -68,8 +68,8 @@ class SolrErrorHandlingSpec extends TestSupportFixture
 
   "delete" should "return the exception bubbling up from solrClient.deleteByQuery" in {
     delete("/fileindex/?q=:") {
-      body shouldBe "Error from server at mockedHost: mocked parser"
-      status shouldBe SC_BAD_REQUEST
+      body shouldBe "solr delete [:] failed with Error from server at mockedHost: mocked parser"
+      status shouldBe SC_INTERNAL_SERVER_ERROR
```

Comment:
why did these status codes change? Perhaps an exception that has changed?

Reply:
I think so, the tests started failing when upgrading `solrj-solrj` from version `7.7.2` to `8.0.0` and higher. However, I can't find information about this change in the [changelogs](https://lucene.apache.org/solr/guide/8_0/major-changes-in-solr-8.html). Do you know if this change would cause other modules interacting with this module to fail?

Reply:
Turns out, there were a number of deprecation warnings in the code. After fixing those, the tests behaved as specified before.